### PR TITLE
[KOGITO-3715] Kogito Operator installer bundle

### PR DESCRIPTION
**Pull request by ricardozanini**
Signed-off-by: Ricardo Zanini <zanini@redhat.com>

See: https://issues.redhat.com/browse/KOGITO-3715

Now the community would deploy the desired Operator version easily without OLM:

```
kubectl apply -f http://github.com/.../v1.0.0/kogito-operator.yaml -n mynamespace
```

@ibek this is my gift for you

@radtriste do you mind taking a look/testing the Jenkins side? I believe I did it right... but who knows.

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-cloud-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains a description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by ricardozanini** [comment]
Tomorrow I update the `install.sh` file.

**Comment by ibek** [comment]
@ricardozanini :) thanks a lot, looking forward to trying it out on K8s

**Comment by ricardozanini** [comment]
LINT error can be ignored for now since this is the first PR to introduce the file.

**Review by Kaitou786 — APPROVED** [review]

File: hack/bump-version.sh [hunk 1]
```diff
@@ -39,7 +39,7 @@ fi
 latest_released_olm_version=$(cd ${OLM_DIR} && for i in $(ls -d */); do echo ${i%%/}; done | grep -v manifests | grep -v ${old_version} | sort -V | tail -1)
 echo "Latest released OLM version = $latest_released_olm_version"
 
-sed -i "s/$old_version/$new_version/g" cmd/kogito/version/version.go README.md version/version.go deploy/operator.yaml ${OLM_DIR}/kogito-operator.package.yaml ${OLM_DIR}/custom-subscription-example.yaml hack/go-build.sh hack/go-vet.sh .osdk-scorecard.yaml
+sed -i "s/$old_version/$new_version/g" cmd/kogito/version/version.go README.md version/version.go deploy/operator.yaml ${OLM_DIR}/kogito-operator.package.yaml ${OLM_DIR}/custom-subscription-example.yaml hack/go-build.sh hack/go-vet.sh .osdk-scorecard.yaml kogito-operator.yaml
```
Comment (radtriste) [inline hack/bump-version.sh:42]: not sure this is needed as you anyway remove the `kogito-operator.yaml` file in the `make vet` command

**Review by radtriste — CHANGES_REQUESTED** [review]
Can you also add a note in README ?

**Review by radtriste — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -34,6 +35,27 @@ Thank you for your interest in contributing to this project!
...
+
+You can quickly install the operator in your local cluster by executing the following command:
+
+```shell script
+NAMESPACE=mynamespace
+VERSION=<current_operator_version>
+kubectl apply -n "${NAMESPACE}" -f "https://github.com/kiegroup/kogito-cloud-operator/releases/download/${VERSION}kogito-operator.yaml"
+```
+
+Replace the values above to match your context. The version information you can grab from the [releases page]().
```
Comment (spolti) [inline README.md:48]: missing the releases link.

**Review by spolti — CHANGES_REQUESTED** [review]
minor thing;

File: README.md (hunk 2 above)
Comment (ricardozanini) [reply README.md:48]: Dammit!

Comment (ricardozanini) [inline README.md:48]:
```suggestion
Replace the values above to match your context. The version information you can grab from the [releases page](https://github.com/kiegroup/kogito-cloud-operator/releases).
```

**Comment by spolti** [comment]
lint fail on kogito-operator.yaml.

**Comment by ricardozanini** [comment]
> lint fail on kogito-operator.yaml.

> LINT error can be ignored for now since this is the first PR to introduce the file.

hehehe

**Pull request merged**